### PR TITLE
31 create movie detail page

### PR DESCRIPTION
**Pull request by TheDoplarEffect**
Movie detail page, 

details on the movie were added with normal API request of Type MovieDetail.  Since we also needed the writer and director a seperate request was made, these values spread onto the first API request and the type Movie detail modified to reflect that.

Cast and crew button will take them to the relevant page once it is merged.

Custom CSS class was needed to get the poster image to fit in a nice way.

### CODE REVIEW

**Review by ReallyBoard123 — CHANGES_REQUESTED** [review]
- [x] spacing between the movie description text
- [x] spacing between castCrewButton and director text
- [x] movie image is being cutoff, check object fit
- [ ] get reservation button not working

**Comment by peelerzin** [comment]
Space between movie description and between castcrew buttons and text applied. Image changed to object-fit. Reservation button redirect to Select Time and Date page.

File: src/routes/Movie.tsx [hunk 1]
```diff
@@ -1,20 +1,107 @@
...
+            >
+              Cast & Crew
+            </Link>
+          </div>
+        </div>
+
+        <div className=" border border-1 border-white-dimmed"> </div>
+        <div className="flex flex-col pt-4 ">
+          <p className="text-white text-primary">Synopsis</p>
+          <p className=" leading-6 text-body text-white-dimmed pt-3">
```
Comment (ReallyBoard123) [inline src/routes/Movie.tsx:89]: you will notice a package called line clamp that has been preinstalled in your project. Please use that to limit the lines to 3

line-clamp-3

**Review by ReallyBoard123 — CHANGES_REQUESTED** [review]
almost there!! This looks great :)

File: src/routes/Movie.tsx (hunk 1 above)
Comment (peelerzin) [reply src/routes/Movie.tsx]: line-clamp-3 added

**Pull request merged**